lib/modules/platform/types.ts

```diff
@@ -43,11 +43,17 @@ export interface RepoParams {
   ignorePrAuthor?: boolean;
 }
 
+export interface PrBodyStruct {
+  hash: string;
+  rebaseRequested?: boolean;
+  reviewableSection?: string;
```

Comment:
Do we use it anywhere?

Reply:
It's mostly for compatibility with https://reviewable.io/, not sure how many repos use it

Reply:
Question, because I have forgotten: were we *adding back* reviewable content when we updated PR bodies, or did we simply strip it before comparing so that we didn't updatePr() every run unnecessarily?

We need to do one of these two things:
- Save the reviewable content if we plan to "put it back", or
- Simply strip it out before calculating the hash, so that it doesn't trip our "pr body needs updating" logic

Reply:
Looking at existing code, I think we simply delete reviewable from our existingPrBody. So we don't need to return `reviewableSection` in this refactor